[2025-05-04T19:29:51.181+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-04T19:29:51.196+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_iceberg_example.dbt_debug manual__2025-05-04T19:29:44.377206+00:00 [queued]>
[2025-05-04T19:29:51.201+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_iceberg_example.dbt_debug manual__2025-05-04T19:29:44.377206+00:00 [queued]>
[2025-05-04T19:29:51.202+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-05-04T19:29:51.211+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_debug> on 2025-05-04 19:29:44.377206+00:00
[2025-05-04T19:29:51.216+0000] {standard_task_runner.py:63} INFO - Started process 1358 to run task
[2025-05-04T19:29:51.218+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_iceberg_example', 'dbt_debug', 'manual__2025-05-04T19:29:44.377206+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/dbt_iceberg_dag.py', '--cfg-path', '/tmp/tmpok6rvjw1']
[2025-05-04T19:29:51.219+0000] {standard_task_runner.py:91} INFO - Job 87: Subtask dbt_debug
[2025-05-04T19:29:51.248+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_iceberg_example.dbt_debug manual__2025-05-04T19:29:44.377206+00:00 [running]> on host 27fc0593f179
[2025-05-04T19:29:51.295+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_iceberg_example' AIRFLOW_CTX_TASK_ID='dbt_debug' AIRFLOW_CTX_EXECUTION_DATE='2025-05-04T19:29:44.377206+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-04T19:29:44.377206+00:00'
[2025-05-04T19:29:51.296+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-04T19:29:51.303+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-04T19:29:51.304+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt_project && dbt debug --profiles-dir .']
[2025-05-04T19:29:51.312+0000] {subprocess.py:86} INFO - Output:
[2025-05-04T19:29:53.285+0000] {subprocess.py:93} INFO - [0m19:29:53  Running with dbt=1.7.0
[2025-05-04T19:29:53.287+0000] {subprocess.py:93} INFO - [0m19:29:53  dbt version: 1.7.0
[2025-05-04T19:29:53.288+0000] {subprocess.py:93} INFO - [0m19:29:53  python version: 3.10.14
[2025-05-04T19:29:53.289+0000] {subprocess.py:93} INFO - [0m19:29:53  python path: /home/***/.local/bin/python
[2025-05-04T19:29:53.289+0000] {subprocess.py:93} INFO - [0m19:29:53  os info: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2025-05-04T19:29:53.393+0000] {subprocess.py:93} INFO - [0m19:29:53  Using profiles dir at .
[2025-05-04T19:29:53.394+0000] {subprocess.py:93} INFO - [0m19:29:53  Using profiles.yml file at ./profiles.yml
[2025-05-04T19:29:53.394+0000] {subprocess.py:93} INFO - [0m19:29:53  Using dbt_project.yml file at /opt/***/dbt_project/dbt_project.yml
[2025-05-04T19:29:53.395+0000] {subprocess.py:93} INFO - [0m19:29:53  adapter type: trino
[2025-05-04T19:29:53.396+0000] {subprocess.py:93} INFO - [0m19:29:53  adapter version: 1.7.0
[2025-05-04T19:29:53.470+0000] {subprocess.py:93} INFO - [0m19:29:53  Configuration:
[2025-05-04T19:29:53.471+0000] {subprocess.py:93} INFO - [0m19:29:53    profiles.yml file [[32mOK found and valid[0m]
[2025-05-04T19:29:53.471+0000] {subprocess.py:93} INFO - [0m19:29:53    dbt_project.yml file [[32mOK found and valid[0m]
[2025-05-04T19:29:53.472+0000] {subprocess.py:93} INFO - [0m19:29:53  Required dependencies:
[2025-05-04T19:29:53.478+0000] {subprocess.py:93} INFO - [0m19:29:53   - git [[32mOK found[0m]
[2025-05-04T19:29:53.478+0000] {subprocess.py:93} INFO - 
[2025-05-04T19:29:53.478+0000] {subprocess.py:93} INFO - [0m19:29:53  Connection:
[2025-05-04T19:29:53.479+0000] {subprocess.py:93} INFO - [0m19:29:53    host: trino
[2025-05-04T19:29:53.480+0000] {subprocess.py:93} INFO - [0m19:29:53    port: 8080
[2025-05-04T19:29:53.481+0000] {subprocess.py:93} INFO - [0m19:29:53    user: ***
[2025-05-04T19:29:53.482+0000] {subprocess.py:93} INFO - [0m19:29:53    database: iceberg
[2025-05-04T19:29:53.483+0000] {subprocess.py:93} INFO - [0m19:29:53    schema: default
[2025-05-04T19:29:53.484+0000] {subprocess.py:93} INFO - [0m19:29:53    cert: None
[2025-05-04T19:29:53.484+0000] {subprocess.py:93} INFO - [0m19:29:53    prepared_statements_enabled: True
[2025-05-04T19:29:53.485+0000] {subprocess.py:93} INFO - [0m19:29:53  Registered adapter: trino=1.7.0
[2025-05-04T19:29:53.511+0000] {subprocess.py:93} INFO - [0m19:29:53    Connection test: [[32mOK connection ok[0m]
[2025-05-04T19:29:53.511+0000] {subprocess.py:93} INFO - 
[2025-05-04T19:29:53.511+0000] {subprocess.py:93} INFO - [0m19:29:53  [32mAll checks passed![0m
[2025-05-04T19:29:54.367+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-04T19:29:54.368+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-04T19:29:54.384+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt_iceberg_example, task_id=dbt_debug, run_id=manual__2025-05-04T19:29:44.377206+00:00, execution_date=20250504T192944, start_date=20250504T192951, end_date=20250504T192954
[2025-05-04T19:29:54.396+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-04T19:29:54.396+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
